CHANGES.rst

```diff
@@ -25,6 +25,7 @@ Unreleased
     to be relative to the containing directory. :issue:`1921`
 -   Completion does not skip Python's resource cleanup when exiting,
     avoiding some unexpected warning output. :issue:`1738, 2017`
+-   Parameterize tests in `test_arguments.py` and `test_basic.py`. :issue:`1828`
```

Comment:
```suggestion
-   Parametrize tests in `test_arguments.py` and `test_basic.py`. :issue:`1828`
```

It is also spelled wrong in the PR title

Reply:
They're both correct. Also, this doesn't require a changelog entry, so I'll remove this anyway.